internal/pkg/manifest/rd_web_svc.go

```diff
@@ -186,6 +191,9 @@ func (s RequestDrivenWebService) applyEnv(envName string) (workloadManifest, err
 
 func (s *RequestDrivenWebService) requiredEnvironmentFeatures() []string {
 	var features []string
+	if s.PrivateEnabled() {
```

Comment:
ahhh this is annoying :homer-disappearing

We need a way to remind ourselves to update this place. On top of my head I'm thinking of this dumb tests...


```go
// In template pkg (unlikely to forget it here because of the `TestEnv_AvailableEnvFeatures` tests
var LBWSEnvironmentFeatures = []string{natgateway, efs, alb..}
var RDWSEnvironmentFeatures = []string{privateservice, natgateway, ...}
```
and here
```go
// In manifest/rd_web_svc_Test.go
func requiredEnvironmentFeatures(t *testing.T) {
   t.Run("all features enabled", func(t *testing.T) {
         svc := RequestDrivenWebService{
...
}
         features, err := svc.requiredEnvironmentFeatures()
         require.EqualElem(features, template.RDWSEnvironmentFeatures)
   })
} 
```

Reply:
i just added a comment - since even with the test way we're just hoping someone notices they need to update a variable ¯\_(ツ)_/¯ lmk if you'd prefer tests though!

Reply:
I think this is good enough for this PR. Even if we do add tests, we will need to do it in a separate PR - was just throwing out ideas 💭